kglab/subg.py

```diff
@@ -436,6 +436,11 @@ def pyvis_style_node ( # pylint: disable=R0201
     style:
 optional style dictionary
         """
+        if notebook == False:
```

Comment:
I have two dubts
* this will fail at runtime if the `notebook` variable is not set? do you expect devs to pass the `notebook` argument to the function? Please provide a unit test.
* condition with boolean should use `is`: `if notebook is False`